mysql-test/suite/sysschema/t/mysqldump.test

```diff
@@ -1,5 +1,7 @@
 -- source include/no_valgrind_without_big.inc
 
+SET @@global.SKIP_SYS_TABLES_ENGINE_CHECK = 1;
```

Comment:
What happens without this?

Reply:
I should add some comments about this. it will fail to create mysql.gtid_executed table due to its engine is innodb while DDSE is rocksdb.